tools/kubernetes/helm/hue/templates/ingress-https.yaml

```diff
@@ -22,10 +23,16 @@ spec:
   - host: {{ .Values.ingress.domain }}
     http:
       paths:
+      {{ if .Values.websocket.enabled }}
+      - backend:
+          serviceName: daphne-websocket
+          servicePort: 8001
+        path: /(ws/.*)
+      {{- end -}}
       - backend:
           serviceName: hue-balancer
           servicePort: 80
-        path: /
+        path: /(.*)
```

Comment:
should we change this in ingress-http as well?

Reply:
We could leave it out for now, WebSockets are still very early and not truly ready. 

This PR is actually more for catching up and avoiding conflicts when modifications are being made to the current helm.